app.py

```diff
@@ -43,6 +48,30 @@
 hyouka_fb_sb = os.environ.get("HYOUKA_FB_SB")
 key_fb_hyouka = os.environ.get("KEY_FB_HYOUKA")
 github_secret = os.environ.get("GITHUB_SECRET")
+username = os.environ.get("USERNAME")
+password = os.environ.get("PASSWORD").encode()
+git_username = os.environ.get("GIT_USERNAME")
+git_password = os.environ.get("GIT_ACCESS_TOKEN")
```

Comment:
`github` rather than `git`.
```suggestion
github_username = os.environ.get("GITHUB_USERNAME")
github_password = os.environ.get("GITHUB_ACCESS_TOKEN")
```